src/util/fields.py

```diff
@@ -1092,20 +1112,27 @@ def _update_field_usage_multi(cr, models, old, new, domain_adapter=None, skip_in
                  WHERE el.name ~ %s
                 """,
                 [r"\y{}\y".format(old)],
-            )
-            fixed_lines_paths = {}
-            for line_id, line_model, line_path in cr.fetchall():
-                new_path = [
-                    new if x.field_name == old and x.field_model in only_models else x.field_name
-                    for x in resolve_model_fields_path(cr, line_model, line_path)
-                ]
-                if len(new_path) == len(line_path) and new_path != line_path:
-                    fixed_lines_paths[line_id] = "/".join(new_path)
-            if fixed_lines_paths:
-                cr.execute(
-                    "UPDATE ir_exports_line SET name = (%s::jsonb)->>(id::text) WHERE id IN %s",
-                    [Json(fixed_lines_paths), tuple(fixed_lines_paths)],
-                )
+            ).decode()
+
+            update_query = "UPDATE ir_exports_line SET name = (%s::jsonb)->>(id::text) WHERE id IN %s"
```

Comment:
Instead of passing the update_query, why not just getting the fixed_paths from the inner function and do the update outside, here directly.